resource/js/translation-service.js

```diff
@@ -0,0 +1,24 @@
+export async function loadLocaleMessages(locale) {
+    const messages = {};
+    try {
+      const response = await fetch(`http://localhost/skosmos/resource/translations/messages.${locale}.json`);
```

Comment:
This didn't work for me, because my local Skosmos installation lives on a slightly different URL. The URL should be changed to a relative one. This worked:

    const response = await fetch(`resource/translations/messages.${locale}.json`);